File: 250315/변수 값 교체하기 6/replacing-variable-values-6.java
public class Main {
    public static void main(String[] args) {
        int a = 2;
        int b = 5;
        int temp = 0;

        temp = a;
        a = b;
        b = temp;

        System.out.printf("%d\n%d", a, b);
    }
}

File: 250315/변수 값 교체하기 6/replacing-variable-values-6.cpp
#include <iostream>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    int a = 2;
    int b = 5;
    
    int temp = a;
    a = b;
    b = temp;

    cout << a << "\n" << b;
}